cdap-security-spi/src/main/java/co/cask/cdap/security/spi/authentication/SecurityRequestContext.java

```diff
@@ -13,9 +13,9 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
-package co.cask.cdap.common.http;
+package co.cask.cdap.security.spi.authentication;
 
-import com.google.common.base.Optional;
+import javax.annotation.Nullable;
 
 /**
  * RequestContext that maintains a ThreadLocal with references to {@code AccessTokenIdentifier}.
```

Comment:
Should we add some more javadocs to this class? Since it is now part of an SPI? Also, how is ``AccessTokenIdentifier`` linked here? Can you check if it produces a javadoc error, since it is part of the ``cdap-security`` module, which should not be available here.